activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb

```diff
@@ -384,7 +384,30 @@ def insert_fixtures(fixtures, table_name)
         manager.into(table)
         columns.each_key { |column| manager.columns << table[column] }
         manager.values = manager.create_values_list(values)
-        execute manager.to_sql, "Fixtures Insert"
+        manager.to_sql
+      end
+
+
+      # Generate and execute a single SQL string that contains multiple INSERT statement for *all* fixtures.
+      # INSERT INTO table (col1, col2) VALUES (?, ?), (?, DEFAULT);\n
+      # INSERT INTO table2 (col1, col2) VALUES (?, ?), (?, DEFAULT);\n
+      #
+      # Overridden in adapters that require something beyond a simple insert (eg. Oracle).
+      def insert_fixtures(fixture_set, tables_to_delete = [])
+        total_sql = String.new
+
+        fixture_set.each do |table_name, fixtures|
+          total_sql << build_sql_for_fixtures_insert(fixtures, table_name)
+          total_sql << ";\n"
```

Comment:
Do we need to consider max_allowed_packet and/or similar settings for other backends?